code/modules/SCP/SCPs/SCP-096.dm

```diff
@@ -214,6 +214,17 @@
 
 /mob/living/simple_animal/hostile/scp096/proc/specialexamine(var/userguy) //Snowflaked.
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/scp096/proc/specialexamine(mob/userguy) //Snowflaked.
```
Why was it a var anyways? Turn the `var/userguy` a few lines up into `mob/userguy` as well. And turn the `..()` into `return ..()`